### DISCUSSION TITLE
Is there a thi-ng debounce?

### DISCUSSION THREAD

**Discussion by cristianvogel** [thi-ng/umbrella#452] [Q&A] [answered]
Was looking if there was a thi-ng debounce...

**Comment by postspectacular** [comment, marked as answer]
Umm, maybe?! :) Can you please give a little bit more context of what you're trying to debounce and in which situ? It's a bit hard to answer otherwise...

For example and for starters, in a reactive stream context ([thi.ng/rstream](https://github.com/thi-ng/umbrella/tree/develop/packages/rstream)), there are:

- https://docs.thi.ng/umbrella/rstream/functions/debounce.html
- https://docs.thi.ng/umbrella/rstream/functions/sidechainPartition-1.html
- https://docs.thi.ng/umbrella/rstream/functions/syncRAF-1.html

In a more general context, the [thi.ng/transducers](https://github.com/thi-ng/umbrella/tree/develop/packages/transducers) package has:

- https://docs.thi.ng/umbrella/transducers/functions/throttle.html
- https://docs.thi.ng/umbrella/transducers/functions/throttleTime.html

**Reply by cristianvogel** [reply]
OK, plenty to dig into there, thanks.
